opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
the chandles are supposed to be identical. If the chandles are not identical, there's a serious bug somwhere.